packages/odyssey-react-mui/src/MenuButton.tsx

```diff
@@ -65,10 +82,25 @@ export type MenuButtonProps = {
    * If the MenuButton is an overflow menu or standard menu.
    */
   isOverflow?: boolean;
+  /**
+   * The horizontal alignment of the menu.
+   */
+  menuAlignment?: "left" | "right";
+  /**
+   * Optional function to fire on button click
+   */
+  onClick?: ButtonProps["onClick"];
+  /**
+   * Optional function to fire on memu close
+   */
+  menuPaperRef?: MutableRefObject<HTMLDivElement>;
+  onClose?: MenuProps["onClose"];
+  preventCloseOnChildClick?: boolean;
   /**
    * The size of the button
    */
   size?: (typeof buttonSizeValues)[number];
+  forceMenuState?: boolean;
```

Comment:
We should use an `is` or `has` prefix. So this would be what? I can't tell what it's doing. We should also add a comment.